conf/cassandra-env.sh

```diff
@@ -267,6 +267,14 @@ JVM_OPTS="$JVM_OPTS -Xlog:gc=info,heap*=trace,age*=debug,safepoint=info,promotio
 # JVM_OPTS="$JVM_OPTS -XX:+UnlockDiagnosticVMOptions -XX:+LogCompilation"
 # JVM_OPTS="$JVM_OPTS -XX:+UnlockCommercialFeatures -XX:+FlightRecorder"
 
+# AVX-512 is still unreliable, so disabling it.
+# https://bugs.openjdk.org/browse/JDK-8292158
+# https://bugs.openjdk.org/browse/JDK-8317121
+# https://mail.openjdk.org/pipermail/hotspot-compiler-dev/2023-September/068447.html
+# Unfortunately, on other platforms like arm, the UseAVX flag is not recognized so we must additionally
+# use IgnoreUnrecognizedVMOptions for Cassandra to startup there.
+JVM_OPTS="$JVM_OPTS -XX:+IgnoreUnrecognizedVMOptions -XX:UseAVX=2"
```

Comment:
do we need to set this in another project for containers as well? (I recall we needed to do that for java agents)